src/controllers/viewCompanyInformation.controller.ts

```diff
@@ -56,4 +70,25 @@ export class ViewCompanyInformationController extends BaseController {
     }
     this.session.setDissolutionSession(this.httpContext.request, updatedSession)
   }
+
+  private validateCompanyDetails(company: CompanyDetails, companyOfficers: DirectorDetails[]): Optional<string> {
```

Comment:
please move into a testable service. Controllers should aim to be as simple as possible.